test/fixture.js

```diff
@@ -1,556 +0,0 @@
-'use strict';
```

Comment:
Hmm, odd. I believe this is a problem with Github and that you will still have a  continuous history for the file although it's not shown in this diff. Because you did a move, you can still do a blame and get information about where each line comes from such as in https://github.com/vega/compass/blame/kw/ts/test/fixture.ts